fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadJob.java

```diff
@@ -305,6 +307,9 @@ public enum JobSubstate {
     @SerializedName("warehouseId")
     protected long warehouseId = WarehouseManager.DEFAULT_WAREHOUSE_ID;
 
+    @SerializedName("pofpe")
+    protected boolean pauseOnFatalParseError = DEFAULT_PAUSE_ON_FATAL_PARSE_ERROR;
```

Comment:
how about persistent it in `jobProperties` like other properties?

Reply:
fixed